src/reverse_keys/views.py

```diff
@@ -13,7 +14,7 @@
 from drf_yasg import openapi
 
 from authorized_keys.models import ReverseServerAuthorizedKeys
-from reverse_keys.utils import issue_token, verify_token, remove_token
+from reverse_keys.utils import issue_token, verify_token, remove_token, get_user_id_from_token
 from reverse_keys.utils import find_multiple_free_ports
 
 from authorized_keys.utils import is_valid_ssh_public_key
```

Comment:
Mark this just to reminds me to remake some better cryptography utilities. : D

Reply:
🥹🥹🥹